packages/react-mutation-mapper/src/component/column/Annotation.tsx

```diff
@@ -57,6 +62,9 @@ export type AnnotationProps = {
     myCancerGenomeData?: IMyCancerGenomeData;
     civicGenes?: RemoteData<ICivicGeneIndex | undefined>;
     civicVariants?: RemoteData<ICivicVariantIndex | undefined>;
+    indexedVariantAnnotations?: RemoteData<
+        { [genomicLocation: string]: VariantAnnotation } | undefined
+    >;
```

Comment:
Looks for all other annotations we pass specific data as a property, like `IHotspotIndex`, `ICivicGeneIdex` instead of a more general `VariantAnnotation`. Should we do the same for vue, something like `IRevueIndex`? Not sure if it is worth the effort though. Maybe it's fine to use the model directly from genome nexus API client.

Reply:
Probably it's fine, we'd like to get this done soon so maybe good to keep the direct model for now